TODO

```diff
@@ -33,10 +33,6 @@ Features:
   with a nice speed-up on services that have many processes running in the same
   cgroup.
 
-* clean up sleep.c:
-  - Make sure resume= and resume_offset= on the kernel cmdline always take
-    precedence
```

Comment:
hmm, what about the "resume=" part of this TODO list item, did that already get fixed?

Reply:
See my comment above: https://github.com/systemd/systemd/pull/12760#pullrequestreview-250605571.

Reply:
@keszybz and @poettering I refactored my patch based on your feedback. This patch has a much smaller footprint on existing code (just a few debug statements added to the existing calculation code) and adds one new function new function called `configure_hibernation` in `sleep.c` which isolates the configuration precedence logic (`/sys/power` before `/proc/swaps`). Look forward to your feedback!

@keszybz I added a more robust commit statement - please let me know if you were hoping to see something different.